src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -214,6 +216,14 @@ public function addConfiguration(NodeDefinition $node)
                         ->end()
                     ->end()
                 ->end()
+            ->end()
+            ->arrayNode('token_verifier')
+                ->beforeNormalization()
+                    ->ifString()->then(function ($v) { return ['service' => $v]; })
+                ->end()
+                ->children()
+                    ->scalarNode('service')->info('The service ID of a custom rememberme token verifier.')->end()
+                ->end()
             ->end();
 
```

Comment:
I would prefer to make this a `token_verifier` scalar node. Making it an array later on (if we want to add more options) is easy to make BC - using the technique you show here.

Reply:
Absolutely makes sense, this was a copy paste from token_provider above sorry :)